front/app/modules/commercial/customizable_homepage_banner/admin/CTASettings.tsx

```diff
@@ -0,0 +1,146 @@
+import { Label } from '@citizenlab/cl2-component-library';
```

Comment:
Refactored version of front/app/modules/commercial/customizable_homepage_banner/admin/CTASettings/index.tsx.